lib/apiservers/portlayer/swagger.yml

```diff
@@ -527,28 +527,129 @@ paths:
           description: "Error"
           schema:
             $ref: "#/definitions/Error"
-  /interaction/{id}/join:
+  /interaction/{id}/resize:
     post:
-      description: "Establish an interaction session with a container by id"
-      summary: "Establishes an interactive session (ala docker attach)"
-      operationId: ContainerJoin
+      description: "Resize the container's tty session"
+      summary: "Resize tty session"
+      operationId: ContainerResize
       tags: ["interaction"]
       consumes:
         - application/octet-stream
+        - application/json
       produces:
         - application/json
       parameters:
         - name: id
           in: path
           type: string
           required: true
+        - name: height
+          in: query
+          type: integer
+          format: int32
+          required: true
+        - name: width
+          in: query
+          type: integer
+          format: int32
```

Comment:
have you tried using: `format: uint32` 

Swagger swagger doesnt support uint so it would fall back to int, but platforms (like go-swagger) that do support uint will us the format as type.